runtime/src/main.ts

```diff
@@ -3,9 +3,8 @@ import { enableProdMode } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
-
-import './assets/stylesheets/shared/main.scss';
-import './assets/stylesheets/planner-overrides.scss';
+//import './../../dist-watch/assets/stylesheets/shared/main.css';
```

Comment:
This is not working properly for runtime... so I did not remove it for reference